device/vr/android/tango/tango_vr_device.cc

```diff
@@ -213,41 +214,58 @@ std::vector<mojom::VRHitPtr> TangoVRDevice::HitTest(float x, float y)
   return mojomHits;
 }
 
-std::vector<mojom::VRPlanePtr> TangoVRDevice::GetPlanes() {
-  std::vector<mojom::VRPlanePtr> mojomPlanes;
-  if (TangoHandler::getInstance()->isConnected())
+mojom::VRPlanePtr CreateMojomPlane(Plane& plane) {
```

Comment:
These method look like they could be static.

Reply:
Done